readthedocs/donate/templates/donate/promo_404.html

```diff
@@ -11,6 +11,7 @@
 {% endblock %}
 
 {% block content %}
+<div style="margin-left: auto; margin-right: auto; width: 900px; text-align: center">
```

Comment:
We shouldn't be putting inline styles into templates, and the width shouldn't be set at a static width here.